### PR TITLE
Updated main menu for touch devices.

### PR DESCRIPTION
**Pull request by adiraomj**
Added check for touch devices using Windows. ( Can anyone please check if this helps https://github.com/siteorigin/siteorigin-north/issues/134 )

Added `pointer:cursor` to `body` if the device being used has iOS.

Main menu looks great on touch devices now. :)

### CODE REVIEW

**Comment by AlexGStapleton** [comment]
Sadly doesn't help with #134. Sorry mate. :(

**Comment by adiraomj** [comment]
Thanks for testing it out @AlexGStapleton. I will send a different PR for that issue.

@Misplon please test out the PR. Let me know if there are any bugs.

**Comment by Misplon** [comment]
@adiraomj Looking smooth man. Just going to ask @gregpriday to give this a quick once over for me and then we'll merge. Thanks.

File: js/north.js [hunk 1]
```diff
@@ -64,29 +64,33 @@ jQuery( function ( $ ) {
 
 	// Remove the no-touch body class for touch devices
 	var isTouchDevice = 'ontouchstart' in document.documentElement;
-	if( isTouchDevice ) {
+	var msTouchEnabled = window.navigator.msMaxTouchPoints;
```
Comment (gregpriday) [inline js/north.js]: Is this second variable needed? Come to think of it, is there a good reason why isTouchDevice is needed? If it's purely for descriptive purposes, then you could just add in comments and put everything in the if statement instead.

```
// This this is a touch device. We detect this through xxxxxx and xxxxx.
if( 'ontouchstart' in document.documentElement || window.navigator.msMaxTouchPoints )
...
```

**Comment by adiraomj** [comment]
As of Internet Explorer 11, the Microsoft vendor prefixed version of this property (msMaxTouchPoints) is no longer supported. It now uses the non-prefixed name, maxTouchPoints.

**Pull request merged**